### ISSUE TITLE
Validation script

### ISSUE THREAD

**Issue by sheaconlon** [sheaconlon/cell_counting#57]
The validation script should do the following.
1. Load the loss metric of the final metric evaluation. Use its data to determine which training step's model had the best loss on the validation set. Load the model from that training step.
2. Evaluate this model's loss over the entire `masks_and_counts` validation set.
3. Evaluate this model's absolute and relative error distributions over the entire `counts_easy` validation set.

A few additional changes are needed.
- The `counts_easy` preprocessing script needs to split `counts_easy` into validation and test sets. This must be done deterministically (verified manually) so that the same test set is withheld from each validation process.
- The training script needs to use the validation set of `counts_easy` to compute absolute and relative error distributions.
- The training script and `masks_and_counts` preprocessing script need to now refer to their test sets as validation sets.

**Comment by sheaconlon** [comment]
Done as of 6b0b4f396bd15cd0c473f6a16d4747fefa64cf84.
